src/main/kotlin/com/google/androidstudiopoet/generators/project/GradlewGenerator.kt

```diff
@@ -29,9 +31,21 @@ object GradlewGenerator {
         val gradlew = "gradlew"
         val gradlewbat = "gradlew.bat"
 
+        if (!File(getAssetsFolderPath()).exists() ||
+                !File(getAssetsFolderPath(), gradlew).exists() ||
+                !File(getAssetsFolderPath(), gradlewbat).exists() ||
+                !File(getAssetsFolderPath(), "gradle/wrapper").exists()
+                ) {
+
+            println("AS Poet needs network access to download gradle files from Github " +
+                    "\nhttps://github.com/android/android-studio-poet/tree/master/resources/gradle-assets " +
```

Comment:
I'm not sure if it is a good idea to reference master. Getting git commit number and referencing it might be a better idea, at least for release.

Reply:
The way how to get git hash is explained here https://stackoverflow.com/a/35041457